Samples/excel-add-in-chart-analyze-data/src/taskpane/taskpane.ts

```diff
@@ -0,0 +1,588 @@
+    });
+  } catch (error) {
+    showStatus(error, true);
+  }
+}
+
+async function showCorrelation2() {
+  try {
+    await Excel.run(async (context) => {
+      showStatus("Running", false);
+      const sheet = context.workbook.worksheets.getItem("Sample");
+
+      let dataRange = sheet.getRange("A40:A41");
+      let chart = sheet.charts.add("XYScatter", dataRange, "auto");
+      chart.name = "correlation2";
+      chart.title.text = "Correlation betweeen Sales and Leaflets";
+      chart.title.left = 8;
+      chart.title.top = 8;
+      chart.title.format.font.size = 12;
+      chart.left = 900;
+      chart.top = 250;
+      chart.width = 300;
+      chart.setPosition("Q2", "V15");
+
+      chart.legend.visible = false;
+
+      let seriesCollection = chart.series;
+      let series = seriesCollection.add("Sales and Leaflets");
+
+      let rangeX = "F2:F33";
+      let xValue = sheet.getRange(rangeX);
+      series.setXAxisValues(xValue);
+
+      let rangeY = "H2:H33";
+      let value = sheet.getRange(rangeY);
+      series.setValues(value);
+
+      seriesCollection.getItemAt(0).delete();
+
+      chart.axes.valueAxis.displayUnit = "Thousands";
+
+      //Show trendline
+      series.trendlines.add("Exponential");
+      let trendline = series.trendlines.getItem(0);
+      trendline.displayRSquared = true;
+
+      //High light title
+      var font = chart.title.getSubstring(21, 5).font;
+      var font2 = chart.title.getSubstring(31, 11).font;
+      font.color = "#FF7F50";
+      font2.color = "#FF7F50";
+
+      await context.sync();
+      showStatus('Success for "Sales/Leaflets correlation"', false);
+    });
+  } catch (error) {
+    showStatus(error, true);
+  }
+}
+
+//Add a ChartTitle to the first chart in the worksheet Sample
+async function showSalesTrendline() {
+  try {
+    await Excel.run(async (context) => {
+      showStatus("Running", false);
+      const sheet = context.workbook.worksheets.getItem("Sample");
+      let dataRange = sheet.getRange("H2:H33");
+      let chart = sheet.charts.add("Line", dataRange, "auto");
+      chart.activate();
+      chart.name = "overallSales";
+      chart.title.text = "Total Sales Trend in September";
+      chart.left = 900;
+      chart.top = 20;
+      chart.width = 300;
+      chart.setPosition("Q16", "V30");
+      chart.legend.visible = false;
+
+      let axis = chart.axes.categoryAxis;
+      let yaxis = chart.axes.valueAxis;
+      yaxis.displayUnit = "Thousands";
+      let categoryNameRange = sheet.getRange("A2:A33");
+      axis.setCategoryNames(categoryNameRange);
+
+      let trendlines = chart.series.getItemAt(0).trendlines;
+      trendlines.add("MovingAverage");
+      let tre = trendlines.getItem(0);
+      tre.movingAveragePeriod = 7;
+      showStatus('Success for "Overall sales trend"', false);
+    });
+  } catch (error) {
+    showStatus(error, true);
+  }
+}
+
+async function clearDatalables() {
```

Comment:
Typo in the function name.
```suggestion
async function clearDataLabels() {
```